build_stage_repository

```diff
@@ -99,15 +99,21 @@ def create_modulemd(collection, version, stage_dir):
     output = check_output(cmd, universal_newlines=True)
 
     with open(modulemd_yaml(collection, version), 'r') as file:
-        modules = yaml.safe_load(file)
+        modules = list(yaml.safe_load_all(file))
+
+    module_version = generate_modulemd_version(version)
+    module_context = generate_modulemd_context(collection, version)
+
+    for module in modules:
+        module['data']['version'] = module_version
+        module['data']['context'] = module_context
+        if module['data']['name'] == collection:
+            module['data']['artifacts'] = {'rpms': output.splitlines()}
```

Comment:
I want to set the version and context for all modules in the file, but artifacts only for the one that is actually our "main" module